webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx

```diff
@@ -59,7 +59,6 @@ export default class DeleteCategoryModal extends React.PureComponent<Props, Stat
                     />
                 )}
                 confirmButtonClassName={'delete'}
-                enforceFocus={false}
```

Comment:
🗒️ 
All instances of enforceFocus with a value of false have been removed. As you can see in the code below, both the Menus and Modals components now handle the focus trap correctly.